### ISSUE TITLE
Exporting plugins from a wrong project

### ISSUE THREAD

**Issue by GoranRista** [davidhealey/hise-exporter#15]
The exporter is exporting plugins from projects that it is not supposed to and may not be in the cue.
For example, let say we have Project A, Project B, and Project C, which are all found under Hise Projects folder. Each project has its own project with the relevant and folder and file hierarchy.
We add only project A to the exporter using HIP file. The project compiles plugins from Project B and places them in the Project A "Compiled" folder.
When this happens, the export cue is interrupted and it stops.
This mix-up also happens when several projects are in the cue. It is rendering Project A, but using files from Project C.
Log is included.

[renderer.log](https://github.com/davidhealey/hise-exporter/files/5805059/renderer.log)

**Comment by GoranRista** [comment]
I am now testing it with XML files and it is compiling the wrong projects still.

**Comment by davidhealey** [comment]
So far I can't recreate the issue so it will be tricky to solve

**Comment by GoranRista** [comment]
Did you put several projects under one "master" folder?

**Comment by davidhealey** [comment]
What do you mean?

**Comment by GoranRista** [comment]
Let's say we have HISE PROJECTS folder. Inside we have:
Project A folder
Project B folder
Project C folder

They are all "one next to each other."

**Comment by davidhealey** [comment]
Yes all my projects are in a similar arrangement

**Comment by GoranRista** [comment]
Uh, that's good that others won't have the same issue, but sucks that I will :) 
If I think of anything, I'll let you know.

**Comment by GoranRista** [comment]
I have discovered that the Exporter is changing the names of my plugins before it renders them as wrong plugins. For example:
Project A plugin gets the name Project C plugin during export. If I open the Project A in HISE and look under Preferences, I see that the name had been changed from Project A to Project C.
So, this is not only a matter of placing wrong files in wrong directories, but actually changing the plugin names. Perhaps the correct plugins get compiled but with wrong names.
I had several projects had their names forcefully chanaged to one particular project's name.
How can we troubleshot this?

**Comment by davidhealey** [comment]
I think we'll need to get in a call and you can share your screen and show me what you're doing

**Comment by GoranRista** [comment]
Yes, that would be good. I think this may have something to do with the last exported project. Earlier I added "Flamenco Percussion" and exported it well. Then when I added "Jazz" it started using Flamenco again. I stopped it, removed everything from the compiling folders and re-initiated the process. Now it appears to be compiling Jazz.
I'll PM you on HISE forum.
